.travis.yml

```diff
@@ -21,6 +21,7 @@ before_script:
 before_install:
   - "npm install -g npm@^4"
   - "google-chrome-stable --headless --disable-gpu --remote-debugging-port=9222 http://localhost &"
+bundler_args: --retry 0
```

Comment:
It is travis specific, depending on provided language travis knows how to build the project, like run `npm install` first and then `npm test` for node projects. It also has some defaults for such projects, e.g. for node to try to run `npm install` 3 times if the first fails, since npmjs sometimes is not available.